docs/network_firewall.rst

```diff
@@ -578,3 +582,25 @@ Once it is complete, you will see a notification of successful upgrade:
 .. [#] Tails screenshots were taken on Tails 3.0~beta4. Please make an issue on
        GitHub if you are using the most recent version of Tails and the
        interface is different from what you see here.
+
+.. _abstract firewall rules:
+
+Abstract firewall rules
+-----------------------
+
+The pfSense instructions using the web interface can also be precisely
+described as follows:
+
+* Disable DHCP (in case the firewall is providing a DHCP server by default)
+* Disallow all traffic by default (inbound or outbound)
+* Allow UDP OSSEC (port 1514) from *Application Server* to *Monitor Server*
+* Allow TCP ossec agent auth (port 1515) from *Application Server* to *Monitor Server*
+* Allow TCP/UDP DNS from *Application Server* and *Monitor Server* to the IP of know name servers
+* Allow UDP NTP from *Application Server* and *Monitor Server* to all
+* Allow TCP any port from *Monitor Server* to all
```

Comment:
TCP outbound on all ports is also needed for the application server - also I would add a specific note in parentheses for both these TCP onbound on all port that it's for Tor, e.g. "this is needed for making connections to the Tor network" since administrators often go off-script on this point.

Reply:
I'm assuming you meant **outbound** instead of **onbound** :-) Added the suggested wording: thanks !